Doc/clarification (#93)
morpho-org/pre-liquidation: README.md

The PreLiquidation smart-contract enforces the properties:

- `preLltv < LLTV`;
- `preLCF1 <= preLCF2`;
- `preLFC1 <= 1`;
- `1 <= preLIF1 <= preLIF2 <= 1 / LLTV`.

Note: A pre-liquidation close factor higher than 100% means that the whole position is pre-liquidatable, so using `preLCF2 > 1` allows to make the position entirely pre-liquidatable at a LTV between preLltv and LLTV.

> The property `preLIF2 <= 1 / LLTV` ensures that pre-liquidation always improves the health of the position.

Reviewer: This is not the only assumption that is needed: health doesn't get worse iff `preLIF <= 1 / LTV`
This is ensured because pre-liquidations are only allowed for `LTV <= LLTV`, which implies that `1 / LLTV <= 1 / LTV`. So together with the property `preLIF2 <= 1 / LLTV` (and `preLIF <= preLIF2`) we get `preLIF <= 1 / LTV`
```suggestion
The property `preLIF2 <= 1 / LLTV` and the fact that pre-liquidations are only allowed for `LTV <= LLTV` ensure that pre-liquidations can't worsen the health of the position.
```

The PreLiquidation smart-contract enforces the properties:

- `preLltv < LLTV`;
- `preLCF1 <= preLCF2`;
- `preLFC1 <= 1`;
- `1 <= preLIF1 <= preLIF2 <= 1 / LLTV`.

> Note: using `preLCF2 > 1`, one can select at which LTV between preLltv and LLTV the entire position can be pre-liquidated.
> A pre-liquidation close factor higher than 100% means that the whole position is pre-liquidatable.

Reviewer 2: ```suggestion
Note: A pre-liquidation close factor higher than 100% means that the whole position is pre-liquidatable, so using `preLCF2 > 1` allows to make the position entirely pre-liquidatable at a LTV between preLltv and LLTV.
```

> The last property ensures that pre-liquidation cannot seize more than the total collateral of the position.

Reviewer 2: ```suggestion
```
misleading imo, this is enforced by Morpho Blue anyway. Plus the close factor is on the debt, not the collat

Author: wanted to justify the property `preLIF2 <= 1 / LLTV` because it's not completely obvious
agree that it's misleading because morpho blue enforce it, i'll try to find a better phrasing otherwise we can drop it

Reviewer 2: oh and also I thought that you where speaking about the last sentence (and not bullet point)

then ok i understand what you mean. We can say to "prevent decreasing the position's health" (that's the precise reason right?)

Author: Just rephrased, yes it's to make sure that preliquidation always improve the health of the position (just noticed that it also ensures that pre-liquidation seize less than the total collateral)

Reviewer 2: why not writing precisely this?
